packages/cli/scripts/ts_prepare.sh

```diff
@@ -0,0 +1 @@
+/home/ajs/zos/packages/lib/scripts/ts_prepare.sh
```

Comment:
was this file committed intentionally?

Reply:
Yes, it is a symlink to a script residing in lib that is very useful for porting.

Reply:
and what about the absolute path?

Reply:
The path is wrong tho... pushing a fix that uses a real file instead.